orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -248,17 +243,29 @@ def to(self, *args, **kwargs):
                 self._device = torch.device(device_str)
         return super(ORTModule, self).to(*args, **kwargs)
 
+    def eval(self: T) -> T:
+        self._is_training = False
+        self._original_module.eval()
+
+    def train(self: T, mode: bool = True) -> T:
+        self._is_training = mode
+        self._original_module.train(mode)
+        if self._is_training and self._device.type == 'cuda':
+            torch.cuda.empty_cache()
```

Comment:
Shouldnt this be called only after export?

Reply:
Correct. This is not needed anymore. 
Updated.